MobileFilter/sections/specific_web.txt

```diff
@@ -76,6 +76,7 @@ rakuten-sec.co.jp###VHEAD > p.ta-c
 rakuten-sec.co.jp##.group-area + p.ta-c
 mamastar.jp##.MstarMB_S_Comp
 dailymail.co.uk##.mpu
+chichi-pui.com#?#.column:has(> div.is-hidden-tablet > script[src*="//js.isboost.co.jp/t/"])
```

Comment:
Script tag will be removed on AG for Android, which will cause the rule to fail. Maybe `#?#.is-hidden-tablet > div.is-mobile > div.column:has(> div.is-hidden-tablet):not(:has(> a))`